_posts/2023-07-05-trino-fest-2023-arcticwolf.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "AWS Athena (Trino) in the cybersecurity space"
+author: "Anas Shakra, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open-table and file formats and
+provides a simplified, flexible way to analyze petabytes of data where it lives.
+Senior software developer Anas Shakra from Arctic Wolf Networks gave a talk at
+[Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %}) detailing
+their switch to AWS Athena and how "queries that took hours with old solution
+now take around a minute today." Tune in to the talk or you can read the recap below!
+
+<!--more-->
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
```

Comment:
Add a sentence about what Arctic Wolf does before this paragraph.

Reply:
Or even better add it to the first paragraph of this blog post about the talk ..